src/components/inputs/TextField.js

```diff
@@ -616,7 +617,7 @@ function createStyles({placeholderTextColor, centered, multiline}) {
     },
     input: {
       flexGrow: 1,
-      textAlign: centered ? 'center' : undefined,
+      textAlign: centered ? 'center' : inputTextAlign,
```

Comment:
You don't need to do that. If you just switch the undefined with 'left', it will turn to 'right' automatically in RTL layout.

Reply:
it actually better when explicitly setting `left` or `right` 
it also moves the cursor of the input to the right (on RTL) 
unfortunately, when just setting `left`, everything works but the cursor

Reply:
Got it. Thanks